apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue

```diff
@@ -191,36 +190,30 @@ const props = withDefaults(defineProps<Props>(), {
 });
 
 const collectorPageStore = useCollectorPageStore();
-const collectorPageState = collectorPageStore.$state;
-
-const { i18nDayjs } = useI18nDayjs();
 
 const storeState = reactive({
     timezone: computed(() => store.state.user.timezone),
 });
 
 const state = reactive({
-    current: computed(() => i18nDayjs.value.tz(i18nDayjs.value(), storeState.timezone)),
-    schedule: computed(() => collectorPageState.schedules.find((schedule) => schedule.collector_id === props.item.collectorId)),
-    scheduleState: computed<CollectorScheduleState>(() => state.schedule?.collector_info.state),
-    toggleStatus: computed(() => (state.scheduleState === COLLECTOR_SCHEDULE_STATE.ENABLED ? 'ON' : 'OFF')),
-    nextSchedule: computed(() => {
-        if (state.schedule) {
-            const hours = state.schedule.schedule?.hours ?? [];
-            const sortedHours = hours.sort((a, b) => a - b);
-            const currentHour = state.current.hour();
-            const closestHour = sortedHours.find((num) => num > currentHour);
-
-            if (closestHour) {
-                return state.current.set('h', closestHour).set('m', 0);
-            }
-            return state.current.set('h', sortedHours[0] ?? 0).set('m', 0).add(1, 'd');
-        }
-        return null;
+    isScheduleActivated: computed(() => {
+        const schedule = props.item.schedule;
+        return !!schedule;
     }),
+    toggleStatus: computed(() => (props.item.schedule.state === COLLECTOR_SCHEDULE_STATE.ENABLED ? 'ON' : 'OFF')),
     diffSchedule: computed(() => {
-        const timeDiff = state.nextSchedule.diff(state.current, 'm');
-        return { diffHour: Math.floor(timeDiff / 60), diffMin: timeDiff % 60 };
+        if (props.item.schedule) {
+            const current = dayjs().utc();
+
+            const userCurrentTime = dayjs.tz(current, storeState.timezone);
+            const hours = props.item.schedule.hours ?? [];
+            const nextSchedule = hours.sort((a, b) => a - b).find((num) => num > userCurrentTime.hour());
+
+            const setNextSchedule = current.set('h', nextSchedule || 0).set('m', 0);
```

Comment:
nitpick: this name feels like that it is a function. What about to change `nextSchedule` to 'nextScheduledHour' and `setNextSchedule` to `nextScheduledTime` ?